docs/release-notes/changelog.rst

```diff
@@ -919,7 +919,7 @@
 
     .. change:: Incorrect ``sync_to_thread`` usage warnings for generator dependencies
         :type: bugfix
-        :pr: 1716, 1740
+        :pr: 1716 1740
```

Comment:
This seems to add 2 commas
![image](https://github.com/litestar-org/litestar/assets/45884264/bf350eea-7627-41ea-ab5b-8b694204deca)

Reply:
Not sure why it behaves like that, but the PR numbers as comma separated list here is intentional.